site/2015/05/05/doctrine-orm-module-release-0-9-0.rst

```diff
@@ -0,0 +1,59 @@
+DoctrineORMModule release 0.9.0
+===============================
+
+The **Zend Framework Integration Team** is happy to announce the new release of **DoctrineORMModule**.
+
+DoctrineORMModule 0.9.0 is out of the door!!
+
+*Note* that this is the last version that supports `doctrine/migrations <https://github.com/doctrine/migrations>`_. We are working on extracting this feature into independent module.
```

Comment:
Last nitpick: `extracting this feature into an independent module` (missed the `an`) ;)